### DISCUSSION TITLE
Rajouter un emplacement d'objet actif pour les objets de combat ou daily

### DISCUSSION THREAD

**Discussion by Chevalier2** [Crownicles/Crownicles#1887] [Idées acceptées / déjà implémentées]
Tout est dans le titre. C'est très embêtant de devoir switch entre daily et combat, ça fait perdre 1h (+30 minutes deux fois car deux switch, daily -> objet combat puis objet combat -> daily), et encore pire si l'on oublie de reprendre l'objet daily (si il est possible de faire le daily, il faudra attendre 30 minutes à cause du switch).

Cet emplacement pourrait remplacer le dernier d'inventaire.
On peut faire ça pour les objets, ou pour les daily, ça fera la même chose.

**Comment by Chevalier2** [comment]
_Ou tout simplement enlever ce cooldown._

**Comment by romain22222** [comment]
j'avoue ne plus utiliser les objets daily, car je les trouve au mieux inutiles, au pire dangereux, car on doit mettre en sursis un de nos objets utiles dans la réserve, et donc potentiellement sans faire exprès en vendre un

**Comment by BastLast** [comment]
J'aime beaucoup la mecanique de devoir choisir entre un objet daily et un avantage en combat, je pense rajouter un système de bonus aux gens qui gardent un objet équipé longtemps.

**Reply by Pietagorh** [reply]
Oui mais nous on l'aime pas >:c

**Reply by romain22222** [reply]
le gros problème de cette mech, ca fait qu'on semble passer à moitié de l'inventaire

**Reply by leuhvraiminoryo** [reply]
évidemment, tu aimes bien ça, car tu joues pas assez pour que ça te fasse inutilement souffrir :slightly_smiling_face: 

tho, le bonus pour ceux qui gardent un objet équipé longtemps, c'déjà cool

**Comment by Chevalier2** [comment]
En fait, non. Exemple concret, j'ai plein d'argent, une bonne guilde, etc. Donc mon objet daily d'argent ne m'est aucunement indispensable. Ce n'est pas pareil pour un nouveau joueur. Donc ce que tu vas faire, c'est désavantager les joueurs les moins haut lvl.

**Comment by BastLast** [comment]
une idée qui résoud la problématique est en cours, go #1888 pour la suite
